### PR TITLE
Delete/updates

### PR DESCRIPTION
**Pull request by YousefAbdelnour**

### CODE REVIEW

File: app/src/Controllers/UpdateController.php [hunk 1]
```diff
@@ -42,4 +43,51 @@ public function handleGetUpdateById(Request $request, Response $response, array
...
+        // Create the new Update
+        $result = $this->updatesService->createUpdate($new_update);
+        $status = $result->isSuccess() ? HTTP_CREATED : 400;
+        $payload = [];
+
+        if ($result->isSuccess()) {
+            //prepare a successful response
+            $payload['status'] = $status;
+            $payload['success'] = true;
+            $payload['inserted_id'] = $result->getData();
```
Comment (cr4zy30) [inline app/src/Controllers/UpdateController.php]: For my POST, PUT, and UPDATE, the way I did it is I return the affected object (inserted_game, updated_game, deleted_game). What do you think?

Something like this:
image

File: app/src/Controllers/UpdateController.php [hunk 2]
```diff
@@ -42,4 +43,51 @@ public function handleGetUpdateById(Request $request, Response $response, array
...
+
+        $status = $result->isSuccess() ? HTTP_OK : 400;
+
+        $payload = [];
+
+        if ($result->isSuccess()) {
+            //prepare a successful response
+            $payload['status'] = $status;
+            $payload['success'] = true;
+            $payload['inserted_id'] = $result->getData();
```
Comment (cr4zy30) [inline app/src/Controllers/UpdateController.php]: Should be deleted_id/deleted_update not inserted

File: app/src/Controllers/UpdateController.php (hunk 1 above)
Comment (YousefAbdelnour) [reply app/src/Controllers/UpdateController.php]: i think its good for updating but not creating, if anything the client can actually try the get and see if it was integrated in the database properly

**Review by cr4zy30 — APPROVED** [review]
😍😍😍😍😍😍😍😍😍

**Pull request merged**